file-validation.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/*
+ * Validation applied to verify wether the bot should execute or skip the file specified
+ */
+
+/**
+ * @param $temp_file_name
+ * @param $file_name
+ *
+ * Validates if the file is valid to be scanned by vip-go-ci
+ *
+ * @return array
+ *	[
+ *		'issues' =>
+ *		[
+ *			'max-lines' => [$file_name],
+ *		],
+ *		'total' => 1
+ *	]
+ */
+function vipgoci_validate( $temp_file_name, $file_name ) {
```

Comment:
I'm more than happy to address this - I'm a "strict type person". I didn't add because I saw most of the functions in this project don't have strict types and wanted to keep consistency).

I also think we can enforce strict_types for the new files/classes at the beginning.